site/content/reference/openshift-permissions.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
I would have this document be more general, not specific to openshift. The permissions are the same no matter the environment. We don't have to mention the SCC specifically, just the permissions that our deployment requires.

Reply:
These would be permissions on the deployment which are in the security context?

Reply:
Yes, plus required volumes